admins/pageflow/accounts.rb

```diff
@@ -3,14 +3,24 @@ module Pageflow
     menu :priority => 3
 
     config.batch_actions = false
-    config.clear_sidebar_sections!
 
     index do
       column :name do |account|
         link_to account.name, admin_account_path(account)
       end
+      column I18n.t('pageflow.admin.accounts.entries_count') do |account|
+        account.entries_count
+      end
+      column I18n.t('pageflow.admin.accounts.users_count') do |account|
+        account.users_count
+      end
+      column I18n.t('pageflow.admin.accounts.default_theming') do |account|
+        account.default_theming.theme_name
```

Comment:
Please override `scoped_scollection` in account admin to prevent N+1 query. See http://activeadmin.info/docs/2-resource-customization.html section "Customizing resource retrieval".